packages/accordion/src/types.ts

```diff
@@ -0,0 +1,13 @@
+import { SlateElement } from '@yoopta/editor';
+
+export type AccordionElementKeys =
```

Comment:
What do you think about using enum here, those you can reuse and reference to it in other places like [here](https://github.com/Darginec05/Yoopta-Editor/pull/135/files#diff-37eadcf5c6abf12de5772c03e095efe7a5db9ea28f84160311cf9413fb372278R11-R16)